src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -76,11 +95,11 @@ const createUiSchemaForString = (ionFormField, remediationForm, transformedResp,
   }
 
   if (Array.isArray(ionFormField.options) && ionFormField.options[0] && ionFormField.options[0].value) {
-    if (shouldRenderAsRadio(ionFormField.name)) {
-      // e.g. { name: 'methodType', options: [ {label: 'sms'} ], type: 'string' | null }
-      uiSchema.type = 'radio';
-      // set the default value to the first value..
-      ionFormField.value = ionFormField.options[0].value;
+    let ionField = ionFormField.options[0];
+    if (ionField.label === 'display') {
```

Comment:
can you confirm radio button can not have `display` as label ? or is it taking care of in `populateUISchemaForDisplay`

Reply:
Hello @pradeepdewda-okta , the `populateUISchemaForDisplay` is taking care of it.